caffe2/core/operator.h

```diff
@@ -587,7 +591,7 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
   }
 
 #if !defined(CAFFE2_IS_XPLAT_BUILD)
-  std::vector<at::Tensor> move_newstyle_outputs() && {
+  c10::ListPtr<at::Tensor> move_newstyle_outputs() && {
```

Comment:
should those be lists actually? those are outputs of an op, not a single gigantic list

Reply:
as discussed offline, this needs to be ListPtr